### DISCUSSION TITLE
[@types/node] `*/promises` modules do not export types

### DISCUSSION THREAD

**Discussion by nickserv** [DefinitelyTyped/DefinitelyTyped#62589] [Issues with a @types package]
For example, I'd expect `import { PathLike } from "fs/promises"` to work because `import { PathLike } form "fs"` works, but the types don't seem to be exported in these modules.

[a comment by an automated account is not shown]

**Comment by SimonSchick** [comment]
Please provide full repro.

**Reply by nickserv** [reply]
https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzgBQIYwBYBlgGsCmcAvnAGZQQhwBEpAzgPRgUjB353VA
